errors.go

```diff
@@ -1,7 +1,61 @@
 package goka
 
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/go-stack/stack"
+)
+
 var (
 	errBuildConsumer = "error creating Kafka consumer: %v"
 	errBuildProducer = "error creating Kafka producer: %v"
 	errApplyOptions  = "error applying options: %v"
 )
+
+var (
+	gokaPackageRegex = regexp.MustCompile(`github.com/lovoo/goka[^/]*/[^/]+$`)
```

Comment:
What if sb. uses a fork? Is there  a smarter way to get the "current" package name?

Reply:
yep, that's exactly the issue. Maybe we really should use reflection like here?
https://golang.org/pkg/reflect/#Type